runserver.sh

```diff
@@ -1 +1,12 @@
-dev_appserver.py --skip_sdk_update_check=yes --port=8080 --admin_port=8000 --host=0.0.0.0 app.yaml skraflstats.yaml
+#!/bin/bash
+
+export GOOGLE_APPLICATION_CREDENTIALS=./resources/Explo\ Development-414318fa79b8.json
```

Comment:
Consider whether this is too much information to have out there in the public (this is a public repo)